test/transforms.jl

```diff
@@ -12,60 +12,60 @@
 
     T = Select(:f, :d)
     n, c = apply(T, t)
-    @test Tables.columnnames(n) == (:f, :d)
+    @test Tables.columnnames(Table(n)) == (:f, :d)
```

Comment:
This shouldn't be necessary. You mean we should return a Vector instead of a Tuple? What the API docs say for Tables.columnnames? Should it return any iterable? Tuples, Vectors, etc.?

Reply:
Resolved in commit af43f28.